### PR TITLE
[VEN-1259] [VEN-1262]: Vault upgrades simulations - mainnet

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Accept the ownership of  VRTVault contracts (previously we had to execute a multisig TX to offer this change) the new owner will be the Normal timelock contract (0x939bd8d64c0a9583a7dcea9933f7b21697ab6396)
Upgrade the implementations of the three vaults

Authorize the Fast-track and Critical timelock contracts to invoke the admin functions in the vaults
**XVSVault**:
> pause
>resume

**VRTVault**
> pause
> resume

**VAIVault**
>pause
>resume

 Authorize the Normal timelock contracts to invoke the admin functions in the vaults, specifically for the following functions:
**XVSVault**
>add(address,uint256,address,uint256,uint256)
>set(address,uint256,uint256)
>setRewardAmountPerBlock(address,uint256)
>setWithdrawalLockingPeriod(address,uint256,uint256)

**VRTVault**
>withdrawBep20(address,address,uint256)

Set the lastAcrruingBlock in VRTVault to 28849755, so after this block no rewards will be accrued in the VRT Vault.

### CODE REVIEW

**Review by kkirka — APPROVED** [review]

**Review by chechu — APPROVED** [review]

File: vips/vip-124.ts [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+const VAI_VAULT_PROXY = "0x0667Eed0a0aAb930af74a3dfeDD263A73994f216";
+const VRT_VAULT_PROXY = "0x98bF4786D72AAEF6c714425126Dd92f149e3F334";
+const XVS_NEW = "0x4d067D2471406a778D103B5742eCC83668ea5345";
+const VAI_NEW = "0x0B4d7776b87deD4a0958E32D5155598057D2620D";
+const VRT_NEW = "0x1405899732Cfe963EaB935a8bBa49c47FE996048";
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
+const MULTISIG = "0x1C2CAc6ec528c20800B2fe734820D87b581eAA6B";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const LAST_ACCRUING_BLOCK = "28849755";
```
Comment (chechu) [inline vips/vip-124.ts:15]:
```suggestion
const LAST_ACCRUING_BLOCK = "29079555";
```

File: vips/vip-124.ts [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    Authorize the Normal timelock contracts to invoke the admin functions in the vaults, specifically for the following functions:
+      XVSVault:
+        add(address,uint256,address,uint256,uint256)
+        set(address,uint256,uint256)
+        setRewardAmountPerBlock(address,uint256)
+        setWithdrawalLockingPeriod(address,uint256,uint256)
+      VRTVault:
+        withdrawBep20(address,address,uint256)
+      
+    Set the lastAcrruingBlock in VRTVault to 28849755, so after this block no rewards will be accrued in the VRT Vault.
```
Comment (chechu) [inline vips/vip-124.ts:49]:
```suggestion
    Set the lastAcrruingBlock in VRTVault to 29079555, so after this block no rewards will be accrued in the VRT Vault.
```

File: simulations/vip-124/simulations.ts [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+const VAI_OLD = "0x7680C89Eb3e58dEc4D38093B4803be2b7f257360";
+const VRT_OLD = "0xA3EEA5e491AD45caE30F6E0a315A275cc99EE147";
+const XVS_NEW = "0x4d067D2471406a778D103B5742eCC83668ea5345";
+const VAI_NEW = "0x0B4d7776b87deD4a0958E32D5155598057D2620D";
+const VRT_NEW = "0x1405899732Cfe963EaB935a8bBa49c47FE996048";
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
+const MULTISIG = "0x1C2CAc6ec528c20800B2fe734820D87b581eAA6B";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const LAST_ACCRUING_BLOCK = "28849755";
```
Comment (chechu) [inline simulations/vip-124/simulations.ts:28]:
```suggestion
const LAST_ACCRUING_BLOCK = "29079555";
```

**Comment by chechu** [comment]
https://app.venus.io/governance/proposal/127

**Pull request merged**